src/UnderlineNav/UnderlineNav.docs.json

```diff
@@ -1,7 +1,7 @@
 {
-  "id": "drafts_underline_nav2",
+  "id": "experimental_underline_nav2",
   "name": "UnderlineNav",
-  "status": "draft",
+  "status": "experimental",
```

Comment:
This is not related to this PR but we should update the status of UnderlineNav to Beta (I'll push a PR)